tests/integration/run.bats

```diff
@@ -23,54 +23,53 @@ function setup() {
 	sudo ip tuntap add tap_nabla_test mode tap
 	sudo ip addr add 21.0.0.1/24 dev tap_nabla_test
 	sudo ip link set dev tap_nabla_test up
-	touch "${BATS_TMPDIR}"/dummy
 }
 
 function teardown() {
 	sudo ip link delete tap_nabla_test
 }
 
 function nabla_run() {
-	run ${TOPLEVEL}/solo5/ukvm/ukvm-bin --net=tap_nabla_test "$@"
+	run ${TOPLEVEL}/rumprun/solo5/tenders/spt/solo5-spt --net=tap_nabla_test "$@"
 
 	echo "nabla-run $@ (status=$status):" >&2
 	echo "$output" >&2
 }
 
 @test "test raw node version" {
-	nabla_run --disk="${BATS_TMPDIR}"/dummy ${TOPLEVEL}/node-base/node.nabla -- '{"cmdline":"node --version"}'
+	nabla_run --x-exec-heap ${TOPLEVEL}/node-base/node.nabla -- '{"cmdline":"node --version"}'
 	[[ "$output" == *"v4.3.0"* ]]
 	[ "$status" -eq 0 ]
 }
 
 
 @test "test raw node (hello)" {
-	nabla_run --disk="${BATS_TMPDIR}"/dummy ${TOPLEVEL}/node-base/node.nabla
+	nabla_run --x-exec-heap ${TOPLEVEL}/node-base/node.nabla
```

Comment:
qn: what's the `--x-exec-heap` flag do? and why is it only used in the node case?

Reply:
It makes the heap executable, it's needed because of Nodejs's JIT compiler. More here: https://github.com/Solo5/solo5/issues/321

Reply:
Have to add it to `runnc` after pointing to latest there as well.

Reply:
yikes... I'm curious about some details here.. Wondering if there are any other situations like this. Let's chat about this at some point!